job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java

```diff
@@ -298,9 +286,10 @@ private synchronized void updateStatus() {
         mPlanInfo.setResult(join(taskInfoList));
         mPlanInfo.setStatus(Status.COMPLETED);
       } catch (Exception e) {
-        mPlanInfo.setStatus(Status.FAILED);
-        mPlanInfo.setErrorType(ErrorUtils.getErrorType(e));
-        mPlanInfo.setErrorMessage(e.getMessage());
+        LOG.warn("Job error when joining tasks Job Id={} Config={} \n {}",
+            mPlanInfo.getId(), mPlanInfo.getJobConfig(),
+            org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace(e));
```

Comment:
You can just let `e` as the last parameter of LOG#warn, the call stack will be printed